ui/k-info-panel/main.kytos

```diff
@@ -0,0 +1,307 @@
+<template>
```

Comment:
@Aduan002 considering that we'll have additional `k-info-panel` when implementing the show maintenance detailed view, what do you think of naming this file `list_maintenance.kytos` or something similar, to facilitate code navigation later on?